tests/test_views_buckets.py

```diff
@@ -247,7 +250,9 @@ def test_every_records_are_deleted_too(self):
                           headers=self.headers)
         self.app.put_json(self.collection_url, MINIMALIST_COLLECTION,
                           headers=self.headers)
-        self.app.get(self.record_url, headers=self.headers, status=404)
+        response = self.app.get(self.record_url, headers=self.headers, status=404)
+        self.assertEqual(response.json['details']['id'], self.record_id)
+        self.assertEqual(response.json['details']['resource_name'], 'record')
```

Comment:
These assertions should be moved to a specific test in `test_views_record.py` that says `test_unknown_record_raises_404`